english/story-continues

```diff
@@ -0,0 +1 @@
+The story continues.  You wake up and it was all a dream.  Then, in the corner of your eye you see [a] (http://ancientexplorers.com/wp-content/uploads/sites/6/2015/12/sparkling-blue-universe_131922.jpg)  
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.